devolutions-gateway/src/config.rs

```diff
@@ -197,7 +203,7 @@ impl Conf {
             sub_provisioner_public_key,
             delegation_private_key,
             plugins: conf_file.plugins.clone(),
-            recording_path: conf_file.recording_path.clone(),
+            recording_path: Some(recording_path),
```

Comment:
With this change, `recording_path` of the `Conf` struct doesn’t need to be an `Option` anymore.